package com.ibm.book.test.exception;

import com.ibm.book.exception.UserNotFoundException;

public class UserNotFoundExceptionTest extends Exception{

}
